pkg/types/installconfig.go

```diff
@@ -294,3 +300,20 @@ type ImageContentSource struct {
 	// +optional
 	Mirrors []string `json:"mirrors,omitempty"`
 }
+
+// CloudCredentialsMode is the mode in which the cloud-credential-operator operates.
+// +kubebuilder:validation:Enum="";mint;passthrough;manual
+type CloudCredentialsMode string
+
+const (
+	// ManualCloudCredentialsMode indicates that cloud-credential-operator should not process any CredentialsRequests.
```

Comment:
https://github.com/openshift/enhancements/blob/master/enhancements/installer/aws-permissions-check-bypass.md#installer

only talks about mint, passthrough and not manual ..

Reply:
+1 would love to see the enhancement updated to match the reality.